types/chai/index.d.ts

```diff
@@ -1109,7 +1109,7 @@ declare namespace Chai {
         property<T>(object: T, property: string, /* keyof T */ message?: string): void;
 
         /**
-         * Asserts that object has a property named by property.
+         * Asserts that object does not have a property named by property.
```

Comment:
This one is an actual incorrect comment, rather than just typos. I got confused by it writing some Chai tests. 😄